### DISCUSSION TITLE
Add support un-used style (lint)

### DISCUSSION THREAD

**Discussion by ngocle2497** [jpudysz/react-native-unistyles#99] [Feature Requests]
Can u add support style un-used (lint error)? like this:
<img width="507" alt="image" src="https://github.com/jpudysz/react-native-unistyles/assets/43195241/512b6c4f-04bb-4077-be08-2fcbd5d5d8a3">

**Comment by jpudysz** [comment]
I can do that. Someone already mentioned it in another thread. There’s no ETA since I prefer contributing to the core. This is more of a nice-to-have rather than something crucial. Perhaps someone from the community will take it up?

**Comment by jpudysz** [comment]
Added to 2.x roadmap
